src/main/java/Duke.java

```diff
@@ -1,10 +1,267 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.function.ToDoubleFunction;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        File dir;
+        File f;
```

Comment:
```suggestion
        File file;
```
Using "file" instead of "f" as the variable name would be preferable, especially since the variable is referenced elsewhere in the code. It can be challenging for people to recall the significance of "f" later on.